pkg/cmd/describe/origin/origin.go

```diff
@@ -1,51 +1,78 @@
-package describe
+package origins
 
 import (
 	"context"
 	"encoding/json"
 	"fmt"
 	"path/filepath"
+	"strconv"
 
 	"github.com/fatih/color"
+	"go.uber.org/zap"
 
 	"github.com/MakeNowJust/heredoc"
 	"github.com/MaxwelMazur/tablecli"
-	msg "github.com/aziontech/azion-cli/messages/origins"
+	msg "github.com/aziontech/azion-cli/messages/describe/origin"
 
 	api "github.com/aziontech/azion-cli/pkg/api/origin"
 	"github.com/aziontech/azion-cli/pkg/cmdutil"
 	"github.com/aziontech/azion-cli/pkg/contracts"
+	"github.com/aziontech/azion-cli/pkg/logger"
 	"github.com/aziontech/azion-cli/utils"
 	sdk "github.com/aziontech/azionapi-go-sdk/edgeapplications"
 	"github.com/spf13/cobra"
 )
 
+var exemplo string = heredoc.Doc(`
```

Comment:
If you want to separate this, I think you could put this var inside the messages file. 
Also: change the var name to "example"

Reply:
I agree, let's put it in the messages to make it cleaner